simulation/sub8_gazebo/nodes/buoys.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+import rospy
+import tf
+
+from gazebo_msgs.msg import ContactsState
+from std_msgs.msg import String
+
+import numpy as np
+
+
+def check_contact(msg, buoy_pub):
```

Comment:
Yep, it'll publish whatever color of the buoy it comes in contact with to `/gazebo/buoy_contact` (I think I named it).